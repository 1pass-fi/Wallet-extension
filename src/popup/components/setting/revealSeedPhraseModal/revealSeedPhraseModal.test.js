import '@babel/polyfill'
import { fireEvent, getByText, render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import React from 'react'
import configureStore from 'redux-mock-store'

import { RevealSeedPhraseModal } from '.'

describe('Test for ImportPhraseLockScreen component', () => {
  let container, onReveal, onClose, setError
  beforeEach(() => {
    setError = jest.fn()
    onReveal = jest.fn()
    onClose = jest.fn(),

    container = render(<RevealSeedPhraseModal
      setError={setError}
      onReveal={onReveal}
      onClose={onClose}
    />).container
  })

  describe('renders without crashing', () => {
    it('renders correctly', () => {
      expect(container).toMatchSnapshot()
    })
  })
})
