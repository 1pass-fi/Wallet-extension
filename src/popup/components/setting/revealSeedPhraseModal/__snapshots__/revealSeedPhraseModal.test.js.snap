// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for ImportPhraseLockScreen component renders without crashing renders correctly 1`] = `
<div>
  <div
    class="modal-container"
  >
    <div
      class="modal reveal-seed-phrase"
    >
      <div
        class="modal-close-icon"
      >
        <svg />
      </div>
      <div
        class="modal-title"
      >
        Reveal Seed Phrase
      </div>
      <div
        class="modal-description"
      >
        If you change browsers or computers, you will need this seed phrase to access your account.
        <br />
        <span>
           
           
        </span>
        Save it somewhere 
        <strong>
          safe and secret
        </strong>
        .
      </div>
      <div
        class="modal-warning"
      >
        <div
          class="warning-icon"
        >
          <svg />
        </div>
        <div
          class="warning-message"
        >
          Never disclose your backup phrase. Anyone with this phrase can steal your accounts.
        </div>
      </div>
      <div
        class="input-field password-field"
      >
        <div
          class="label"
        >
          Enter your password
        </div>
        <div
          class="input-wrapper"
        >
          <input
            class="input"
            name=""
            placeholder=""
            type="password"
            value=""
          />
          <svg
            class="toggle-display"
          />
        </div>
      </div>
      <div
        class="buttons-line"
      >
        <button
          class="button-shared button-filled modal-action-button reveal"
        >
          Reveal
        </button>
        <button
          class="button-shared button-outline modal-action-button close"
        >
          Go Back
        </button>
      </div>
    </div>
  </div>
</div>
`;
